org.eclipse.sprotty.xtext/src/main/java/org/eclipse/sprotty/xtext/ls/DiagramUpdater.xtend

```diff
@@ -38,17 +38,17 @@ import com.google.inject.Singleton
 @Singleton
 class DiagramUpdater {
 
-	//@Inject 
-	DiagramLanguageServer languageServer 
+	//@Inject
+	protected DiagramLanguageServer languageServer
```

Comment:
For consistency with the field `resourceServiceProviderRegistry` I'd rather add a `@Accessors(PROTECTED_GETTER)` annotation. And while we're at it, you could also remove the commented-out `@Inject`.

Reply:
Didn't catch that in the first commit, it is now added in this PR.